modules/oe_paragraphs_timeline/oe_paragraphs_timeline.post_update.php

```diff
@@ -29,3 +30,22 @@ function oe_paragraphs_timeline_post_update_00001(array &$sandbox) {
 
   return t('The heading field has been created for Timeline paragraph.');
 }
+
+/**
+ * Add optional introduction field to Timeline paragraph.
+ */
+function oe_paragraphs_timeline_post_update_00002(): TranslatableMarkup {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_paragraphs_timeline') . '/config/post_updates/00002_introduction_field');
+
+  $config_record = $storage->read('field.field.paragraph.oe_timeline.field_oe_text_long');
+  $field = FieldConfig::load($config_record['id']);
```

Comment:
We know the id, why not to be explicit? So you don't need to load up $config_record variable just yet, just to check if the field exists already or not, would be better to load up the config once we are going to use it.